flang/lib/Semantics/check-omp-structure.h

```diff
@@ -223,6 +225,9 @@ class OmpStructureChecker
       const parser::OmpObjectList &ompObjectList);
   void CheckPredefinedAllocatorRestriction(
       const parser::CharBlock &source, const parser::Name &name);
+  void checkAndAddSymbolsToUsedInScanList(const parser::OmpObjectList &x);
+  void addModifiertoMap(const parser::OmpObjectList &x,
+      parser::OmpReductionClause::ReductionModifier &modifier);
```

Comment:
Nit: Follow capitalization conventions on this file. Also, I think the name for one of these methods can be improved, but feel free to name it differently from what I'm suggesting if you think it's better.
```suggestion
  void CheckAndRegisterScanSymbols(const parser::OmpObjectList &x);
  void AddModifierToMap(const parser::OmpObjectList &x,
      parser::OmpReductionClause::ReductionModifier &modifier);
```